src/popup/components/AccountInfo/index.tsx

```diff
@@ -43,7 +43,7 @@ class AccountInfo extends Component<WithStyles & IProps, {}> {
           <Typography className={classes.token}>QTUM</Typography>
           {hasRightArrow && <KeyboardArrowRight className={classes.rightArrow} />}
         </div>
-        <Typography className={classes.balanceUSD}>{qtumBalanceUSD}</Typography>
+        <Typography className={classes.balanceUSD}>{qtumBalanceUSD}{networkMarkup}</Typography>
```

Comment:
nit: use String template and add a space between qtumBalanceUSD and networkMarkup

Reply:
are you saying to do this? 
<Typography className={classes.balanceUSD}>`{qtumBalanceUSD} {networkMarkup}`</Typography>
the ` ` show up on the page when I do that